apps/graphql/src/apps/itinerary/queries/SearchOneWayItineraries.js

```diff
@@ -27,7 +27,7 @@ const ItinerariesOneWay = {
     args: Args<ItinerariesOneWaySearchParameters>,
     { dataLoader }: GraphqlContextType,
   ) => {
-    const itineraries = await dataLoader.itineraries.load(args.input);
+    const itineraries = await dataLoader.itineraries_new.load(args.input);
```

Comment:
I believe that `itineraries_new` is just temporary name and will be renamed to `itineraries` in the future, right?

Reply:
Yep, for now we keep these two until the migration to the new schema is done